app/scripts/controller-init/snaps/snaps-registry-init.ts

```diff
@@ -0,0 +1,31 @@
+import { JsonSnapsRegistry } from '@metamask/snaps-controllers';
+import { ControllerInitFunction } from '../types';
+import { SnapsRegistryMessenger } from '../messengers/snaps';
+
+/**
+ * Initialize the Snaps registry controller.
+ *
+ * @param request - The request object.
+ * @param request.controllerMessenger - The messenger to use for the controller.
+ * @param request.persistedState - The persisted state of the extension.
+ * @returns The initialized controller.
+ */
+export const SnapsRegistryInit: ControllerInitFunction<
+  JsonSnapsRegistry,
+  SnapsRegistryMessenger
+> = ({ controllerMessenger, persistedState }) => {
+  const requireAllowlist = Boolean(process.env.REQUIRE_SNAPS_ALLOWLIST);
```

Comment:
IIRC the env variables are replaced directly with booleans?

Reply:
Not according to TypeScript at least. Either way, `Boolean(true)` is the same as `Boolean('true')`

Reply:
Yeah, but `Boolean('false') === true`. Can we assert the type of it?